facebook/delphiFacebook/R/utils.R

```diff
@@ -148,7 +149,7 @@ megacounty <- function(
   df_megacounties <- mutate(df_megacounties,
                             geo_id = make_megacounty_fips(.data$geo_id))
 
-  df_megacounties <- group_by(df_megacounties, .data$day, .data$geo_id)
+  df_megacounties <- group_by_at(df_megacounties, groupby_vars)
```

Comment:
> Are you using it just for the ability to take a character vector of column names, instead of needing symbols?

Yes. I'll look into alternatives. I believe I'm using another `*_at` function elsewhere also.

Reply:
I ended up using `group_by(across(<character vector>))` so existing calls to `megacounty()` don't need to be modified.

It looks like `write_data_api()` is also using a superseded `*_at` function [here](https://github.com/cmu-delphi/covidcast-indicators/blob/77274a1ccc1c7fcd5950313242d40ee7ebae23ce/facebook/delphiFacebook/R/to_api.R#L28). I don't see a straightforward way to change this to `mutate`, which doesn't seem to support functional modification of a set of columns.